priv/riak_kv.schema

```diff
@@ -967,3 +967,45 @@
   fun(Value) when is_integer(Value) andalso Value > 0 -> true;
      (_) -> false
   end}.
+
+%% @doc Register allowable origins (for CORS), specified as a list of ip
+%% ({host} or {host, port} tuple) under a scheme, i.e.
+%%
+%% {riak_kv, [
+%%   {allowable_origin, {
+%%      http, [
+%%          {"127.0.0.1", 3000},
+%%          {"10.10.1.240"}
+%%      ],
+%%      https, [
+%%          {"127.0.0.1", 4430}
+%%      ]
+%%   }}
+%% ]}
+%%
+%% The resulting allowable origins are "http://127.0.0.1:300",
+%% "http://10.10.1.240:*", "https://127.0.0.1:4430".
+%%
+%% The special case of "Allow All" is achieved by specifying the scheme as '*'
+%% (wildcard), i.e.
+%%
+%% {riak_kv, [
```

Comment:
It's an `ip` data type, and cuttlefish afaik doesn't allow for wildcards in that